config/routes.rb

```diff
@@ -40,6 +40,8 @@ def matches?(request)
 
       get "/resources/:account" => "resources#index"
 
+      get "/resources" => "resources#index"
+
```

Comment:
Seems like `/resource/:kind` could be useful here, too.

Reply:
I assume you mean `/resources/:kind`; then it would clash with the account route. I suppose if it was `/resource/:kind` (singular) then it wouldn't clash, but it would be a bit unexpected.

It's a good idea to add a route qualified only with a kind, but I'm not sure how would that look to make sense. `/resource/:kind` is weird because it's not just one resource; I almost wish we had `/resources/:kind` in the first place, and `/accounts/:account/resources/...` would then qualify by account. Of course we could merge the routes and have `/resources/:account_or_kind`, but the meaning would depend on the state of the system and that'd _really_ be confusing.

Hmm... what do you think about a wildcard placeholder, eg. `/resources/_/:kind`?

Reply:
I think `/resources?kind=foo` suffices

Reply:
Yeah, I meant `/resources/:kind`.

@kgilpin's suggestion that we use a query string works for me.

Reply:
Yes, it does.